nautobot/core/settings.py

```diff
@@ -536,6 +625,10 @@
 
 CELERY_BEAT_SCHEDULER = "nautobot.core.celery.schedulers:NautobotDatabaseScheduler"
 
+# When `True` an attempt to queue a duplicate task will raise a `DuplicateTaskError`. The default
+# behavior is to return the `AsyncResult` for the existing task.
+CELERY_SINGLETON_RAISE_ON_DUPLICATE = True
```

Comment:
Is this still relevant in the current diffs? I don't see any other reference to this variable.